src/envo/envo-edit.owl

```diff
@@ -44974,6 +44976,27 @@ AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#has
 AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/ENVO_03605005> "epixylon"@en)
 SubClassOf(<http://purl.obolibrary.org/obo/ENVO_03605005> <http://purl.obolibrary.org/obo/ENVO_03605000>)
 
+# Class: <http://purl.obolibrary.org/obo/ENVO_03605006> (freshwater stream)
+
+AnnotationAssertion(<http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/ENVO_03605006> "A stream where fresh water flows.")
+AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#created_by> <http://purl.obolibrary.org/obo/ENVO_03605006> <ORCID:0000-0001-9076-6066>)
```

Comment:
same issue as before - please use the full URL and not a literal but an IRI/URI in the appropriate field. You can also set this up to add your ORCID automatically as described here: https://github.com/EnvironmentOntology/envo/wiki/Adding-classes-to-ENVO

Reply:
thanks for providing the links